Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM ruby:2.6.2-slim-stretch
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev default-libmysqlclient-dev nodejs-legacy curl git
+FROM ruby:3.0.0-slim-buster
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev default-libmysqlclient-dev nodejs libnode64 curl git
```

Comment:
`nodejs-legacy` was available up to debian streatch, but not from debian buster.
This image use `nodejs` and `libnode64` instead.